### PR TITLE
feat: image& input

### PR DESCRIPTION
**Pull request by bhtibrewal**
in this pr: 
- Added image and input components

### CODE REVIEW

File: src/components/image/image.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  width: 100%;
+  object-fit: contain;
+}
+
+.img-round {
+  border-radius: 50%;
+  aspect-ratio: 1/1;
+}
+.img-round img {
+  width: 100%;
```
Comment (Shivani-1524) [inline src/components/image/image.css:18]: you could reduce the size of the rounded image, its a bit too big.

File: src/components/input/input.html [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+        <!-- input w/icon -->
+        <div class="component-desc">
+            <h3>Input with Icon</h3>
+            <p>Add a icon and give class <span class="highlight-text">icon-input</span> to the div. </p>
+        </div>
+        <div class="component-container ">
+            <div class="artsy-input icon-input" >
+                <span class="input-label">search</span>
+                <i class="fas fa-search"></i>
```
Comment (Shivani-1524) [inline src/components/input/input.html:88]: the search icon would look a lot better in the right end of the search field.
Also, your input fields look really good.

File: src/components/input/input.html [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+    <!-- main -->
+    <main>
+        <h1>Input</h1>
+        <p class="body-l"></p>
+
+        <!-- inputs -->
+        <div class="component-desc">
+            <h3>Text Input</h3>
+            <p>By default artsy ui inputs are outline styled, to change to standard style input add class <span class="highlight-text">input-std</span> to the input.<br>
+            To mark a input as req just add class <span class="highlight-text">input-req</span> to the div. </p>
```
Comment (Shivani-1524) [inline src/components/input/input.html:62]: try to not use abbreviations in descriptions of components. here, "req" could be changed to "required"

**Review by Shivani-1524 — CHANGES_REQUESTED** [review]

File: src/components/image/image.html [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+        <div class="component-container">
+            <div class="artsy-img">
+                
+                <img src="https://picsum.photos/300/200" alt ="random images" loading="lazy" />
+            </div>
+            <div class="artsy-img img-responsive">
+                <img src="https://picsum.photos/300/200" alt ="random images" loading="lazy" />
+            </div>
+
```
Comment (Naman-Saxena1) [inline src/components/image/image.html:89]: The problem of using img-responsive like this is the initial intrinsic dimensions of image is small (300-200) and then when you do width: 100% in css it blurs, that is why srcset is a good option. In srcset attribute you provide img set and browser chooses best image according to available dimensions.

Other option is you use high resolution image as original image and then work with its dimensions.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]

**Comment by gauravsinhaweb** [comment]
images can be fixed other than that, the input field looks good

nice work!

**Comment by AshutoshM29** [comment]
Looks good to me! You can merge it. Nice work♥️

**Pull request merged**